managers/HotelManager.py

```diff
@@ -0,0 +1,59 @@
+from generals.SQLHandler import SQLHandler as DB
+from generals.Error import Error
+from generals.QueryType import QueryType
+from models.HotelRoom import HotelRoom
+
+
+class HotelManager:
+    # Check if hotel id exists
+    @staticmethod
+    def hotel_exists(hotel_id):
+        hotel_query = "SELECT hotel_id FROM hotels WHERE hotel_id = {0}" \
+            .format(str(hotel_id))
+        try:
+            return DB.execute_query(hotel_query, QueryType.Single) is not None
+        except Error as err:
+            raise err
```

Comment:
Docs in python are written like this:
```suggestion
    @staticmethod
    def hotel_exists(hotel_id):
        """"
        Check if hotel id exists
        """
        hotel_query = "SELECT hotel_id FROM hotels WHERE hotel_id = {0}" \
            .format(str(hotel_id))
        try:
            return DB.execute_query(hotel_query, QueryType.Single) is not None
        except Error as err:
            raise err
```

Reply:
fixed